shell/plugins/plugin.js

```diff
@@ -18,5 +20,57 @@ export default async(context) => {
     }
   });
 
-  await allHash(hash);
+  // Discover all of the UI Plugins CRs
+
+  let loadPlugins = true;
+
+  // Provide a mechanism to load the UI without the plugins loaded - in case there is a problem
+  if (context.route?.path === '/safeMode') {
```

Comment:
Do we need to consider the deployed case of `/dashboard/safeMode`, or will `dashboard` be excluded because it's our root path?

Reply:
Yes - good spot - fixed